src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -51,16 +51,16 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 MyExtensibilitySolutionService.Instance.GetService(GetType(IMenuCommandService)),
                 IMenuCommandService)
             Debug.Assert(vsMenuService IsNot Nothing, "Could not get vsMenuService!")
-            listViewExtensions.MenuCommandService = vsMenuService
+            DataGridViewExtensions.MenuCommandService = vsMenuService
 
             RefreshExtensionsList()
 
-            ' Resize each columns based on its content.
-            If listViewExtensions.Items.Count > 0 Then
-                For i As Integer = 0 To listViewExtensions.Columns.Count - 1
-                    listViewExtensions.AutoResizeColumn(i, ColumnHeaderAutoResizeStyle.ColumnContent)
-                Next
-            End If
+            For Each column As DataGridViewColumn In DataGridViewExtensions.Columns
+                column.AutoSizeMode = DataGridViewAutoSizeColumnMode.AllCells
+                column.Width = column.Width
```

Comment:
This line *shouldn't* have any effect.  If it does, then it needs a comment about why it's required.

Reply:
Fixed, it was left after I was testing some things.